.github/workflows/codeql-analysis.yml

```
@@ -63,9 +63,8 @@ jobs:
     #    and modify them (or add more) to build your code if your project
     #    uses a compiled language
 
-    #- run: |
-    #   make bootstrap
-    #   make release
+    - run: |
+       make RELEASE=1 RUNTESTS=0
```

```suggestion
       make RELEASE=1 RUNTESTS=0 -j2
```
GitHub Action runners have 2 cores [reference: https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners]